ansible/playbook.yml

```diff
@@ -50,5 +53,6 @@
     environment:
       CI_TOKEN: "{{ ci_token }}"
       PULL_SECRET_PATH: "{{ pull_secret_remote_path }}"
+      OCP_VERSION: "{{ ocp_version | default('') }}"
```

Comment:
Maybe better to set here hard coded default OCP version, instead of empty string
So the default behavior is to use this hard coded version instead of the latest

Reply:
The problem with setting a default value is that we will have to chose one now and also maintain it later